presto-main/src/test/java/com/facebook/presto/cost/TestCoefficientBasedStatsCalculator.java

```diff
@@ -45,7 +45,7 @@ public TestCoefficientBasedStatsCalculator()
 
         queryRunner.createCatalog(
                 queryRunner.getDefaultSession().getCatalog().get(),
-                new TpchConnectorFactory(1, true),
```

Comment:
> In regards of testing for case with predicate pushdown enabled
and disabled, one can select columns which either support or do not
support predicate according to test needs.

I kind of disagree. The set of columns that have support for predicate push down is allowed to grow and it will grow, e.g. when someone writes a test for push down of predicates on two columns.

Reply:
It is true, but in regards of TPCH the set of columns where you could pushdown the predicate is limited to dictionary like columns.

Reply:
What about changing the default, instead of removing the switch?

Reply:
`There is no practical reason that one could prefer predicate pushdown
disabled.`

Switch makes things more complicated and here I do not see added value, but I only see a cost.

Reply:
`There is no practical reason that one could prefer predicate pushdown
disabled.`

Switch makes things more complicated and here I do not see added value, but I only see a cost.

For example because of this switch, predicate pushdown practically was wrongly implemented for about year and nobody noticed that.

Reply:
Yea, but what about testing how Presto behaves when a connector doesn't have predicate push down? I am worried that we're losing test coverage here. Of course, we're also gaining coverage of tpch connector itself, and also for how Presto behaves when a connector _does_ have predicate push down... This is why I propose to retain the switch.

Reply:
> In regards of testing for case with predicate pushdown enabled
and disabled, one can select columns which either support or do not
support predicate according to test needs.

TPCH supports predicate pushdown for two tables. So this is a kind of hybrid connector which can be used for both kind of tests.